lua/plenary/path.lua

```diff
@@ -73,7 +73,7 @@ local _split_by_separator = (function()
 end)()
 
 local is_uri = function(filename)
-  return string.match(filename, "^[%w+-.]://") ~= nil
+  return string.match(filename, "^%w+[%w\\+\\-\\.]*://") ~= nil
```

Comment:
I'm sorry for my previous PR which introduced this issue. I just noticed the error on my machine and found your PR.

Considering the first character can only be a letter (which is `%a`) and the next letters can be any combination of letters, digits, plus, period or hyphen, shouldn't it be like this?
```suggestion
  return string.match(filename, "^%a[%w+-.]*://") ~= nil
```

Reply:
Never mind and thanks for correcting me. that's just make it alright.